tests/4.5/target_teams_distribute/test_target_teams_distribute_device.c

```diff
@@ -61,7 +63,7 @@ int main() {
     for (int x = 0; x < ARRAY_SIZE; ++x) {
       OMPVV_TEST_AND_SET_VERBOSE(errors[dev], a[x] != 1 + dev + b[x]);
       if (a[x] != 1 + dev + b[x]) {
-	break;
+        break;
       }
     }
   }
```

Comment:
Split up the array into sections for each device to work on to be mapped back non-overlapping.

Reply:
This change has been made and is passing all summit compilers.